src/core/foundations/src/typography/types.ts

```diff
@@ -67,6 +69,6 @@ export type FontScaleFunctionStr = (options?: FontScaleArgs) => string
 export interface FontScaleArgs {
 	lineHeight?: LineHeight
 	fontWeight?: FontWeight
-	italic?: boolean
```

Comment:
Can we keep the old `italic` argument alive for now, to avoid introducing a breaking change?

Reply:
If both are present does the new argument take priority?

Reply:
Yeah, I guess that makes sense! Hopefully nobody will do that if we delete the documentation for the old API 😂

Reply:
Ok, should be updated now